ds/data/processed/migration/net_migration_rate_internal.nuts2.yaml

```diff
@@ -0,0 +1,31 @@
+api_doc_url: First sheet in https://www.ons.gov.uk/file?uri=%2fpeoplepopulationandcommunity%2fpopulationandmigration%2fmigrationwithintheuk%2fdatasets%2flocalareamigrationindicatorsunitedkingdom%2fcurrent/lamis2020final.xlsx
```

Comment:
This need to be interpreted as a URL as it goes as is in the correspoedent link in the info modal. It'd add the note on the sheet in `description_long` which get shown in the info modal too so that the user will see it in context.